src/css/styles.css

```diff
@@ -1,3 +1,74 @@
+body { padding-top: 0px; }
```

Comment:
you are overwriting a lot of bootstrap styles as far as I can see, this can break layout of other modules. Please use BEM naming conventions if you want to override styles.